### PR TITLE
Add contribution guide

### PR DESCRIPTION
**Pull request by JannisMilz**
Created a clear contribution guide. Everything covered you need to start working on the codebase.

**Would need someone to read trough the whole guide, even try some things and check if you understand it. (Can open in branch to see final version)**

Issue: #3

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+## 💪 Other Ways to Help
+
+### Sending Feedbacks & Reporting Bugs
+
+If you had any issues, bugs, or want to share about your experience, feel free to do so on our GitHub issues page or at our [Discord](https://discord.gg/GjJNf7zhgy).
+
+### Submitting New Ideas
+
+If you think AquaHub could use a new feature, please open an issue on our GitHub repository, stating as much information as you can think about your new idea and it's implications. We would also use this issue to gather more information, get more feedback from the community, and have a proper discussion about the new feature.
```
Comment (AtomicCoconut) [inline CONTRIBUTING.md:167]: I would add a link to the issue site.

Comment (AtomicCoconut) [reply CONTRIBUTING.md:167]: "please open an [issue](https://github.com/AquaHubStudio/AquaHub/issues)", something like that.

**Review by AtomicCoconut — COMMENTED** [review]
Looks good 👍🏻

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+│       ├── components # UI components (also in ../routes if only on one page needed)
+│       ├── guards # Guards to protect from accessing different pages
+│       ├── helpers # Functions for additional functionality
+│   ├── routes # Svelte routes
+├── static # Statically served files
+│   ├── assets # Necessary assets
+├── tests # UI tests
+```
+
+## ✅ Tests
```
[the remark this answers is not in the document]
Comment (AtomicCoconut) [reply CONTRIBUTING.md:155]: It just looks a bit odd.

File: CONTRIBUTING.md (hunk 1 above)
Comment (JannisMilz) [reply CONTRIBUTING.md]: Goog point. Also changed it at the top of the file.

**Review by AtomicCoconut — APPROVED** [review]
👍🏻

**Pull request merged**